stories/topics/proc-gcp-deploying-extension-nodes.adoc

```diff
@@ -11,6 +11,12 @@ You configure extension nodes after you have purchased and launched an extension
 . In the *Zone* field, select a zone in the *Region* you selected when deploying your foundational offer. 
 This field is only used to filter the Network list.
 . In the *Main Deployment Name* field, enter the foundation deployment name for which you are deploying an extension.
++
+[NOTE]
+====
+*Main Deployment Name* is required field and it is not treated as optional field. 
```

Comment:
is a required field. 

if it's a required field the end of the sentence about it not being optional is not needed.